dev/benchmarks/microbenchmarks/lib/foundation/all_elements_bench.dart

```diff
@@ -42,7 +42,10 @@ Future<void> main() async {
     ),
   ));
 
-  await SchedulerBinding.instance.endOfFrame;
+  // Wait for frame rendering to stabilize.
+  for (int i = 0; i < 5; i++) {
+    await SchedulerBinding.instance.endOfFrame;
+  }
```

Comment:
Are we guaranteed to get five frames?

Reply:
In test runs this appears to work.  I'm not sure what would be safest here given that AFAIK there isn't a signal from the framework indicating when enough frames have rendered.

Reply:
int > + for (i= 5, i=0, i++)
int ++(int 5, int 0, i=<4)
devcpp frames await working does it shoes it powerfull to get rendering of powerfull energy and senelc frames in senegal down country shop and country sheet excel, powerpoint and powerfull static that names me mouhamadou lamine e3dprod emonsite call back general sic 
i=4; i= 4; i= 5; int = 4 and i traines 5 or 4 if 5 render 5 
decepit correction giving to mster powerfull shetll sheet on computing system names e3dprod sic return back code on cppdroid and droidedit frames on c++droid c mobile apkpure and store jqure and smartphone business company

Reply:
How about this?

```suggestion
  assert(SchedulerBinding.instance.hasScheduledFrame);
  for (int i = 0; i < 5 && SchedulerBinding.instance.hasScheduledFrame; i++) {
    await SchedulerBinding.instance.endOfFrame;
  }
```

Reply:
`SchedulerBinding.instance.hasScheduledFrame` is false at this point in the script

Reply:
Ahh actually, looking at endOfFrame's impl, it looks like it schedules a frame if one isn't already pending. Ok. This should be fine.